README.rst

```diff
@@ -0,0 +1,107 @@
+courseraoauth2client
+====================
+
+.. image:: https://travis-ci.org/coursera/courseraoauth2client.svg
+    :target: https://travis-ci.org/coursera/courseraoauth2client
+
+This project is a library consisting of a command line interface and a client
+for interacting with Coursera's OAuth2 authorized APIs.
+
+Installation
+------------
+
+To install this sdk, simply execute::
+
+    sudo pip install courseraoauth2client
+
+`pip <https://pip.pypa.io/en/latest/index.html>`_ is a python package manager.
+If you do not have ``pip`` installed on your machine, please follow the
+installation instructions for your platform found at:
+https://pip.pypa.io/en/latest/installing.html#install-or-upgrade-pip
+
+Setup
+-----
+
+Before using Coursera's OAuth2 APIs, be sure you know your client id,
+client secret, and scopes you want for for your application. You may create
+an application at https://accounts.coursera.org/console. When creating the
+application, set the
+``Redirect URI`` to be ``http://localhost:9876``.
+
+Then put your client id, client secret, and scopes
+in a file at ``~/.coursera/courseraoauth2client.cfg``::
```

Comment:
Yep. That makes sense. I've tried to change the configuration such that every app that needs to be used is configured separately, and the cache for that app is stored separately. Here's a minimal example of multiple apps using this library (after setting both of them up): https://gist.github.com/cliu587/17a616a4ef2eea13fc2263f1d7d19177